events/notifications.py

```diff
@@ -26,16 +29,26 @@ class Labels:
 
 
 def send_event_created_notification(event):
-    email = event.contract_zone.get_contact_email()
-    if not email:
+    contact_email = event.contract_zone.get_contact_email()
+    if contact_email:
+        send_notification(
+            contact_email,
+            NotificationType.EVENT_CREATED,
+            {"event": event, "user": event.contract_zone.contractor_user},
+        )
+    else:
         logger.warning(
-            'Contract zone {} has no contact email so cannot send "event created" notification.'.format(
+            'Contract zone {} has no contact email so cannot send "event created" notification there.'.format(
                 event.contract_zone
             )
         )
-        return
 
-    send_notification(email, NotificationType.EVENT_CREATED, {"event": event})
+    for official in User.objects.filter(is_official=True):
```

Comment:
can ilmoitin's [`admins_to_notify`](https://github.com/City-of-Helsinki/django-ilmoitin/blob/develop/django_ilmoitin/models.py#L29) be somehow used for this? user should have `is_staff=True` to be eligible for that. If `is_official` does not mean `is_staff`, then I guess, not.